### ISSUE TITLE
All zero targets?

### ISSUE THREAD

**Issue by fferroni** [r0nn13/conditional-dcgan-keras#4]
https://github.com/r0nn13/conditional-dcgan-keras/blob/master/conditional_gan.py#L193

This creates an array of all zeros. Surely half should be zero and the other half ones?

**Comment by lazysquid** [comment]
Yeah I think same.
I think it should be 
```python
y = np.concatenate(np.ones(10,1,64,64), np.zeros(10,1,64,64))
```

**Comment by kristosh** [comment]
Furthermore, what is the reason of input the y in line 194. Since in the loss function we do not take into account y but we use the labels provided from the loss function itself. Isn't pointless? Secondly, why the y is a vector of size 20x1x64x64? What is the reason for the 64?
